auth/src/main/java/com/firebase/ui/auth/ui/phone/CountryListSpinner.java

```diff
@@ -86,6 +86,7 @@ public CountryListSpinner(Context context, AttributeSet attrs, int defStyle) {
             public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
                 CountryInfo info = mCountryListAdapter.getItem(position);
                 if (info != null) {
+                    mSelectedCountryInfo = info;
                     setText(info.toShortString());
```

Comment:
nit: Maybe replace these two lines with a call to the `setSelectedForCountry` helper method?